src/pykoala/ancillary.py

```diff
@@ -28,10 +28,9 @@ def vprint(*arg, **kwargs):
     """
     Prints the arguments only if verbose=True.
     """
-    if kwargs['verbose']:
+    if 'verbose' in kwargs:
```

Comment:
Shouldn't we check that `verbose` is `True`?
```python
    if 'verbose' in kwargs and verbose is True:
```
or
```python
    if kwargs.get('verbose', False):
```

Reply:
`'verbose'` is just a string, not a boolean. The old line asks if `kwargs['verbose']` is `True`, not if 'verbose' is `True`, which would be a nonsense thing to ask. The new line (whether or not 'verbose' is an element in the list of keys `kwargs`)  states the same condition without spitting out a `KeyError`